Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
 
 COPY runtime/tests/google-chrome.repo /etc/yum.repos.d/google-chrome.repo
 
-RUN yum install -y google-chrome-stable
+RUN yum install -y google-chrome-beta
```

Comment:
So, we'll never run tests using stable chrome henceforth?

Reply:
@rgarg1 I'm not sure what difference beta and stable have. We could revert to the stable version later on. Right now the installation channel for `chrome-stable` is broken. We cannot install latest stable version of chrome via the official channel.

Reply:
@jarifibrahim Well Beta as the name suggests is beta quality! Do I need to say more?
Accepting this PR as a temporary fix.